code-challenges.js

```diff
@@ -30,8 +52,13 @@ var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "tem
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
 // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
-
-
+// Create a function that finds only numbers
+const onlyNums = (arr) => {
+// return only numbers, remember typeof will help here
+    return arr.filter(value => typeof value === 'number')
+}
+// again do not forget to run mixedDataArray inside of your function
+console.log(onlyNums(mixedDataArray))
```

Comment:
I'd look back into the requirements of the challenge. While you were able to filter the numbers from the other datatypes, it is NOT sorting from least to greatest! 

Be sure to DOUBLE CHECK your requirements and check your expected output versus your actual output.